### PR TITLE
ISPN-14803 Make sure block hound works in a server

### PR DESCRIPTION
**Pull request by wburns**
https://issues.redhat.com/browse/ISPN-14803

### CODE REVIEW

**Comment by wburns** [comment]
I need to rework some more of the Blockhound integrations to get a server running.

**Comment by wburns** [comment]
I was able to run a server on main and was able to complete a RESP benchmark, we can tweak more as needed after this PR.

File: commons/all/src/main/java/org/infinispan/commons/internal/CommonsBlockHoundIntegration.java [hunk 1]
```diff
@@ -59,5 +65,28 @@ private static void handleJREClasses(BlockHound.Builder builder) {
...
+      // Allow logging to block
+      builder.allowBlockingCallsInside(java.util.logging.Logger.class.getName(), "log");
+   }
+
+   private static void log4j(BlockHound.Builder builder) {
+      try {
+         Class.forName("org.apache.logging.log4j.spi.AbstractLogger");
+         builder.allowBlockingCallsInside(AbstractLogger.class.getName(), "logMessage");
+      } catch (ClassNotFoundException e) {
+         throw new RuntimeException(e);
```
Comment (jabolina) [inline commons/all/src/main/java/org/infinispan/commons/internal/CommonsBlockHoundIntegration.java:87]: Should it fail without the AbstractLogger?

Comment (wburns) [reply commons/all/src/main/java/org/infinispan/commons/internal/CommonsBlockHoundIntegration.java:87]: Oh whoops, let me fix that.

**Comment by wburns** [comment]
Should be good now.

**Comment by wburns** [comment]
Note that the blockhound jar has to be added to the lib directory and cannot be installed in the server/lib directory due to how our class loading works.

**Review by jabolina — APPROVED** [review]

**Pull request merged**